### DISCUSSION TITLE
Show chevron down icon in <select multiple> field

### DISCUSSION THREAD

**Discussion by novasash** [orchidjs/tom-select#263] [Q&A] [answered]
Hello,

What is the best way to have the chevron down icon show up in the `<select multiple>` type of field the way it's showing for a `<select>` field? Need this functionality for a Rails app that's using TomSelect.

Thank you!

**Comment by oyejorge** [comment, marked as answer]
There are a lot of different possibilities but it depends on your CSS or SCSS. If you're using bootstrap4, [here's the scss](https://github.com/orchidjs/tom-select/blob/master/src/scss/tom-select.scss#L86). If you're using bootstrap5, [it's this scss](https://github.com/orchidjs/tom-select/blob/master/src/scss/tom-select.bootstrap5.scss#L215).

**Reply by novasash** [reply]
I used `.single` class. Thank you @oyejorge!
